db/migrate/20200329171111_add_index_to_status_in_sessions.rb

```diff
@@ -0,0 +1,5 @@
+class AddIndexToStatusInSessions < ActiveRecord::Migration[5.2]
```

Comment:
since the sessions table is a new table we can do some Rails magic and let the `create_table` add an index for us. See https://github.com/coronagoapp/coronago-backend/pull/16/files#r399832965